README.md

```diff
@@ -1 +1,54 @@
-# Club in a Box
+# HackEDU's Amazing Readme
+
+## Step 1: You have issues, I have issues, we all have issues!
+Every Github project has issues. Literally.
+
+### Issues == Reddit_posts  and Issues == Facebook_posts
+An issues has a topic, some content, and can be commented upon just like a post on many social networks. An issues made for a github project is typically about a bug, recommendation, or relevant discussion for the project. Issues can also have tags and can be search for. Old issues (that have been resolved or are no longer relevant) are closed, which simply means that they no longer appear when searching for open threads. They are still archived and can even be reopened if needed. If you need help anywhere in this tutorial just open up an issue and we'll get right on it. 
+
+### Lets talk with issues
+In [project name] we use issues for any discussion relevant to running a club. If you have a question, proposal, or something you want to say of general interest, just open up an issue!
+
+## Step 2: Contribute!
```

Comment:
Do we want this section in `CONTRIBUTING.md`, so github will display it on PR/issue submission pages? ![](https://camo.githubusercontent.com/71995d6b0e620a9ef1ded00a04498241c69dd1bf/68747470733a2f2f6769746875622d696d616765732e73332e616d617a6f6e6177732e636f6d2f736b697463682f6973737565732d32303132303931332d3136323533392e6a7067)